Management/src/ActuatorWeb/OpenTelemetryExtensions.cs

```diff
@@ -0,0 +1,72 @@
+using System.Text.RegularExpressions;
+using OpenTelemetry;
+using OpenTelemetry.Context.Propagation;
+using OpenTelemetry.Instrumentation.AspNetCore;
+using OpenTelemetry.Instrumentation.Http;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Resources;
+using OpenTelemetry.Trace;
+using Steeltoe.Common;
+using B3Propagator = OpenTelemetry.Extensions.Propagators.B3Propagator;
```

Comment:
That's a neat trick to hide the obsolete warning I didn't know was possible. But instead of working around the warning, can we follow the suggested alternative instead?

> [Obsolete("Use B3Propagator class from OpenTelemetry.Extensions.Propagators namespace, shipped as part of OpenTelemetry.Extensions.Propagators package.")]

Reply:
Both projects are already referencing the new package, this seems to be required to do what they're suggesting

Reply:
Ah, I get it now.